python_modules/dagster/dagster/core/definitions/sensor_definition.py

```diff
@@ -387,8 +399,9 @@ def check_valid_run_requests(self, run_requests: List[RunRequest]):
         if run_requests and not self._targets:
             raise Exception(
                 f"Error in sensor {self._name}: Sensor evaluation function returned a RunRequest "
-                "for a sensor without a specified target. Targets can be specified by providing "
-                "a job or pipeline_name."
+                "for a sensor lacking a specified target (pipeline_name, job, or jobs). Targets "
```

Comment:
would like to see tests for these error messages

Reply:
added